### ISSUE TITLE
Mering plugins?

### ISSUE THREAD

**Issue by pcav** [desertwolf7/NumericalDigitize#1]
Is this a version of the old Numerical Digitize, ported to QGIS3?
If so, better adding a new version to the existing repository than creating a new one.
I'd appreciate if you could clarify.
Cheers.

**Comment by desertwolf7** [comment]
Yes.
When I worked at QGIS 2.x I actively used plugin NumericalDigitize. When I decided to switch to QGIS 3 I discovered that this plugin was missing. The author has not been active for five years. Then I decided to write the plugin myself. As a basis I took the source code of the old plugin and significantly refactored and expanded it.
The functionality of the new plugin covers the functionality of the two old plugins - 'NumericalDigitize' and 'Numerical Vertex Edit' and complements them with new capabilities. But it requires QGIS 3.4+.
Best regards
